scripts/docker/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM alpine
+
+RUN apk add --no-cache bash libstdc++ gmp-dev libc6-compat && ln -s libcrypto.so.1.1 /lib/libcrypto.so.10
+
+EXPOSE 9000/tcp 14555/tcp 6000/tcp
```

Comment:
can we add some comment here about the port? 

9000 is the base port of harmony node
14555 is the RPC server port
6000 is the REST server port?